### DISCUSSION TITLE
How to properly set specific file and backup folders in a single S3 bucket?

### DISCUSSION THREAD

**Discussion by Tresmos** [pocketbase/pocketbase#2520] [Q&A] [answered]
I've been trying to configure Pocketbase to use specific folders inside a single bucket for file and backup storage for multiple instances like this:
```
pocketbase/
  pbData1/
  pbBackup1/
  pbData2/
  pbBackup2/
```
but I can't seem to figure it out. I don't know if it is possible currently. I did get it to work by adding bucket name to the end of the endpoint like `http://minio:9000/pocketbase` and in the bucket field, setting the folder name I want. It seems to work even though the admin ui gives an error like this:

```
Failed to test the S3 filesystem. Raw error: 
failed to delete a test file: blob (code=NotFound): NoSuchKey: The specified key does not exist. status code: 404.
```
Uploading and getting files seems to work but deleting does not and backups list doesn't populate at all.

Is there a way to properly configure Pocketbase like this? If so a little nudge in the right direction would be appreciated.

**Comment by ganigeorgiev** [comment, marked as answer]
I guess this is related to #237. This can't be done at the moment. It is recommended to use a single bucket per application (_or ideally 2 separate - one for the uploaded files storage and another for the backups; also make sure that you have the correct ACL permissions for your backup bucket to avoid accidentally making the archives public_).

**Reply by Tresmos** [reply]
I see. Yeah using different buckets per application is probably the way to go for critical data but for non critical hobby projects, having an option would be nice. Thanks for the answer!
